pkg/kubelet/apis/kubelet.config.k8s.io/helpers_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package kubeletconfig
+package kubelet
```

Comment:
this is the drawback with using dots in the pkg name -- it's not obvious what the default package name is

Reply:
This is a valid criticism; tradeoff between picking the default package name and having a directory name consistent with the api group name.

Reply:
Added a commit that changes things back to kubeletconfig